connectors/spark/src/main/scala/io/unitycatalog/spark/auth/GenericCredentialProvider.java

```diff
@@ -87,11 +96,6 @@ public GenericCredential accessCredentials() {
     return credential;
   }
 
-  // For testing purpose only.
-  void setRenewalLeadTimeMillis(long renewalLeadTimeMillis) {
```

Comment:
Removed this for testing purpose `setter`, since we are passing it via the `constuctor`.